app/utils/ViewUtils.scala

```diff
@@ -30,10 +30,7 @@ object ViewUtils {
   private def titleNoForm(title: String, section: Option[String], titleMessageArgs: Seq[String])(implicit
     messages: Messages
   ): String =
-    s"${messages(title, titleMessageArgs: _*)} - ${section.fold(emptyString)(
-        messages(_) +
-          " - "
-      )}${messages("service.name")} - ${messages("site.govuk")}"
+    s"${messages(title, titleMessageArgs: _*)}${section.fold(emptyString)(sec => s" - ${messages(sec)}")}"
```

Comment:
Could you please write tests for this change as part of ViewUtilsSpec (we should create one as it does not exist currently)?